src/api/apiCore.cpp

```diff
@@ -361,6 +361,13 @@ rgl_graph_get_result_size(rgl_node_t node, rgl_field_t field, int32_t* out_count
 		CHECK_ARG(node != nullptr);
 
 		auto pointCloudNode = Node::validatePtr<IPointsNode>(node);
+		if (!pointCloudNode->isLastExecOk()) {
```

Comment:
I get the motivation, understand the problem, and only want to point, that I hate this part of the code here.
Definitely leave TODO here.